lib/pathresolver.js

```diff
@@ -39,6 +39,9 @@ PathResolver.prototype = {
       for (var j = 0, attr; j < constants.URL_ATTR.length; j++) {
         attr = constants.URL_ATTR[j];
         attrValue = dom5.getAttribute(node, attr);
+        if (attrValue === '') {
```

Comment:
This is actually valid assetpath if imported doc is in same folder.  The problem is the logic that checks whether to rewrite it is using truthiness of the value of empty string both here and in the two rewrite functions.  The '.' value better represents the relative path needed in the rewrite functions, so I set it here to solve all three uses.  However, setting assetpath itself to '.' is not desirable.